haste/windowed_conformal_model/conformal_interestingness_model.py

```diff
@@ -125,11 +125,24 @@ def interestingness(self,
             sum_of_intensities_ts_features = time_series_features(sum_of_intensities_ndarray,
                                                                   timestamps_ndarray,
                                                                   timestamp)
+
             # .. = (mean,sd,d1,d2)
 
-            # TODO: concatenate with same ts-series features for correlations
+            #Correlation
+            correlation = course_features[0]
+            correlation_ndarray = numpy.ndarray(shape=(len(correlation)),
+                                                       dtype=int,
+                                                       buffer=numpy.array(correlation))
+
+            # Compute features on the entire timeseries so far for that well:
+            correlation_ts_features = time_series_features(correlation_ndarray,
+                                                                  timestamps_ndarray,
+                                                                  timestamp)
+
+            ts_features = numpy.append(sum_of_intensities_ts_features, correlation_ts_features, axis=0)
```

Comment:
maybe print this to the console (just to check that its sensible - and the correct data type?)